Kernel/FourVelocity.m

```diff
@@ -9,55 +9,86 @@
 
 
 BeginPackage["KerrGeodesics`FourVelocity`",
-	{"KerrGeodesics`ConstantsOfMotion`"}];
+	{"KerrGeodesics`ConstantsOfMotion`",
+	"KerrGeodesics`OrbitalFrequencies`"}];
 
 KerrGeoFourVelocity::usage = "KerrGeoVelocity[a,p,e,x] returns the four-velocity components as parametrized functions.";
 
 Begin["`Private`"];
 
 
-(* ::Subsection::Closed:: *)
```

Comment:
Can you make sure to close the sections before saving? Having these changed mixed in with other actual changes makes it harder to review.